lms/djangoapps/grades/rest_api/v1/gradebook_views.py

```diff
@@ -281,10 +282,19 @@ def get(self, request, course_key):
                 'assignment_types': self._get_assignment_types(course),
                 'subsections': self._get_subsections(course, graded_only),
                 'grades_frozen': are_grades_frozen(course_key),
-                'can_see_bulk_management': can_see_bulk_management(course_key),
+                'can_see_bulk_management': self._can_see_bulk_management(course_key),
             }
             return Response(results)
 
+    def _can_see_bulk_management(self, course_key):
+        """
+        Whether or not to show bulk management for this course. Currently, if a course has a
+        master's track or is enabled with the grades.bulk_management course waffle flag.
+        """
+        course_modes = get_course_enrollment_details(str(course_key), include_expired=True).get('course_modes', [])
```

Comment:
`include_expired=True` follows existing behavior, but it might be worth confirming behavior with Product. Removing this would mean a course could lose access when a master's track closes.

Reply:
This feels test-able, and test-worthy

Reply:
I think it should remain, and I agree it should be tested